root.tf

```diff
@@ -16,6 +16,19 @@ module "github_preservica_client_repository" {
   }
 }
 
+module "github_log4j_plugin_repository" {
```

Comment:
Should this have "sbt assembly" in the name like `github_sbt_assembly_log4j_plugin_repository`?

Reply:
Good point, I've changed it.